cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -50,6 +50,7 @@ THE SOFTWARE.
             </j:forEach>
         </d:tag>
     </d:taglib>
+    <f:entry title="Workflow DSL Reference" help="/${it.DSL_REF_URL}"/>
```

Comment:
> Not sure what you mean by swapping, though.

Well `dslHelp` is really the reference. And `dslReference` is what you see when you click the help button. So the view names seem backwards.